scss/mixins/_grid-framework.scss

```diff
@@ -14,10 +14,12 @@
           flex-basis: 0;
           flex-grow: 1;
           max-width: 100%;
-          min-height: 1px;
           padding-right: ($gutter / 2);
           padding-left:  ($gutter / 2);
         }
+        .col-#{$breakpoint}:empty {
+          min-height: 1px;
```

Comment:
Would you mind re-adding this explanatory comment while you're at it?:
https://github.com/twbs/bootstrap/blob/7c86bc9664ddc3a1965fe91d7a0db97eb9348448/less/mixins/grid-framework.less#L19